docs/docs/configuration.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: Configuration
+nav_order: 2
+permalink: /docs/configuration/
+---
+
+# Configuration
+
+By default, Spotty is looking for a `spotty.yaml` file in the root directory of the project. This file describes 
+parameters of a remote instance and an environment for the project. Here is a basic example of such file:
+
+```yaml
+project:
+  name: my-project-name
+  syncFilters:
+    - exclude:
+      - .git/*
+      - .idea/*
+      - '*/__pycache__/*'
+
+container:
+  projectDir: /workspace/project
+  image: tensorflow/tensorflow:latest-gpu-py3
+  ports: [6006, 8888]
+  volumeMounts:
+    - name: workspace
+      mountPath: /workspace
+
+instances:
+  - name: i1
+    provider: aws
+    parameters:
+      region: eu-west-1
+      instanceType: p2.xlarge
+      volumes:
+        - name: workspace
+          parameters:
+            size: 10
+```
+
+## Available Parameters
+
+Configuration file consists of 4 sections: `project`, `container`, `instances` and `scripts`.
+
+### __`project`__ section:
+
+- __`name`__ - the name of your project. It will be used to create S3 bucket and CloudFormation stack to run 
+an instance.
+
+- __`syncFilters`__ _(optional)_ - filters to skip some directories or files during synchronization. By default, all 
+project files will be synced with the instance. Example:
+    ```yaml
+    syncFilters:
+      - exclude:
+          - .idea/*
+          - .git/*
+          - data/*
+      - include:
+          - data/test/*
+      - exclude:
+          - data/test/dump.json
+    ```
+    
+    It will skip ".idea/", ".git/" and "data/" directories except the "data/test/" directory. All files from 
+    the "data/test/" directory will be synced with the instance except the "data/test/dump.json" file.
+    
+    You can read more about filters 
+    here: [Use of Exclude and Include Filter](https://docs.aws.amazon.com/cli/latest/reference/s3/index.html#use-of-exclude-and-include-filters){:target="_blank"}. 
+
+### __`container`__ section:
+
+- __`projectDir`__ - a directory inside the container where the local project will be copied. If
+it's a subdirectory of a container volume, the project will be located on that volume,
+otherwise the data will be lost once the instance is terminated.
+
+- __`image`__ _(optional)_ - the name of the Docker image that contains environment for your project. For example, 
+you could use [TensorFlow image for GPU](https://hub.docker.com/r/tensorflow/tensorflow/){:target="_blank"} 
+(`tensorflow/tensorflow:latest-gpu-py3`). It already contains NumPy, SciPy, scikit-learn, pandas, Jupyter Notebook and 
```

Comment:
For Jupyter to be included you need to use image with *-jupyter tag. See https://hub.docker.com/r/tensorflow/tensorflow/tags (there where som changes from 1.12)

Reply:
Thanks, @jagin, updated it.